### PR TITLE
Add first 3 test cases for Project Sharing

### PR DESCRIPTION
**Pull request by bdattoma**
This PR is adding 2 Smoke and 1 Sanity tests for Project Sharing feature coming in v1.27:
- ODS-2194
- ODS-2201
- ODS-2202

### CODE REVIEW

**Comment by bdattoma** [comment]
> Should we fix the Robocop alearts ?

done!

**Comment by bdattoma** [comment]
First PR test: [1752](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1752/TestReport/). Almost OK, there is a minor issue with a check. Fixed and a new run is going on.

Second PR test: [1753](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1753/console) _RUNNING_

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+*** Variables ***
+${SAVE_BUTTON}=    xpath://button[@data-id="save-rolebinding-button"]
+${PERMISSIONS_DROPDOWN}=    xpath://td[@data-label="Permission"]//button[@aria-label="Options menu"]
+
+
+*** Keywords ***
+Assign ${permission_type} Permissions To ${username}
+    [Documentation]    Ass the user ${username} and level of permission ${permission_type}
```
Comment (jgarciao) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource:13]: I think you mean _Assigns_  :-)

**Review by jgarciao — APPROVED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource (hunk 1 above)
Comment (bdattoma) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource:13]: LOL

Comment (bdattoma) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource]: fixed

**Review by diegolovison — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**